README.md

```diff
@@ -63,6 +63,8 @@ PATH="$HOME/Projects/buildkite/agent:$PATH" ./script/update-agent-help.sh
 
 We spell-check the docs (American English) and run a few automated checks for repeated words, common errors, and markdown and filename inconsistencies.
 
+To run most of these checks, run `./scripts/vale.sh`.
```

Comment:
```suggestion
You can run most of these checks with `./scripts/vale.sh`.
```

Reply:
@karensawrey Not going to fight this, but could you tell me about your thinking here? I'm often inclined to write things in the pattern of "To achieve outcome X, do Y" (or more substantially, "To achieve outcome X:" followed by a procedure—inspired significantly by [the MS guidance on this stuff](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions)). I would like to know how to better follow the house style.

Reply:
@ddbeck my thinking is very simple - I have to have two "run"s in a single sentence. 
"To **run** most of these checks, **run** `./scripts/vale.sh`."